README.markdown

```diff
@@ -1,5 +1,31 @@
 # What is GitX?
 
+GitX is an OS X (MacOS) Native graphical client for the `git` version control system.
+
+
+GitX has a long history of various branches and versions maintained by various people over the years.  This github org & repo are an attempt to consolidate and move forward with a current, common, community-maintained version.   See discussion at [**Forking, and plans for the future of GitX**](https://github.com/gitx/gitx.github.io/issues/1).
```

Comment:
Agreed.  Just trying to do the bare minimum to have something current in the README